### ISSUE TITLE
Allow HTTP Redirect based payment integration

### ISSUE THREAD

**Issue by theArsalanM** [mirumee/saleor#5341]
### What I'm trying to achieve
I'm trying to integrate SagePay Form integration to saleor, but seems like current payment backend doesn't allow HTTP redirect based payment handling.

### Describe a proposed solution
There should be flexibility of custom payment integration where card details are entered entirely at payment gateways portal, where user is redirected along with order details in the form of HTTP post redirect.

### Sample Integration
https://www.sagepay.co.uk/support/find-an-integration-document/form-integration

**Comment by srbraun** [comment]
I am also interested in same functionality. Any advice on how it could be implemented?

**Comment by petedermott** [comment]
Might be worth noting that SagePay offer a REST API (although they really hide [the docs](http://qa-developer-portal-home.sageaccountantscloud.com/api/payments/) for it)

It will take a lot more work to integrate but it is possible to do with Saleor

**Comment by theArsalanM** [comment]
Yes, SagePay offers API but some payment gateways only support redirect based payment.
Also, with redirect based implementation developers don't need to worry much about payment compliances, as payment gateway takes care of it.

This feature would definitely give some more freedom to developers.

**Comment by Zagrebelin** [comment]
Is there any news on this issue?

**Comment by maarcingebala** [comment]
Recommended ways of implementing payments are now via webhooks/apps, which are more flexible than Saleor plugins, and allow for redirect-based payments.
